README.md

```diff
@@ -71,7 +71,7 @@ Many systems provide pre-built packages:
 * macOS via [Homebrew](https://brew.sh/):
 
   ```
-  brew install --cask osxfuse
+  brew install --cask macfuse
   brew install s3fs
```

Comment:
When I run the second command it says:

> Error: s3fs has been disabled because it requires closed-source macFUSE!

Could you join the discussion in #1618?  This suggests another path forward.

Reply:
yeah will do.

Reply:
Just so this is documented somewhere, osxfuse had a free license and macfuse replaces it with a less-free license.  The former is unmaintained.  Does this match your understanding?

Also could you remove the unrelated commits from this PR and submit them separately?